Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.8
+// swift-tools-version: 5.9
```

Comment:
This bumps support to the 5.9 release of Swift Package Manager. `Package@swift-5.8.swift` is there in case someone is still using Xcode 14, which they otherwise wouldn't be able to use.